core/src/main/scala/akka/persistence/cassandra/query/AllPersistenceIdsStage.scala

```diff
@@ -31,9 +31,9 @@ import scala.concurrent.ExecutionContextExecutor
  */
 @InternalApi private[akka] final class AllPersistenceIdsStage(
     refreshInterval: Option[FiniteDuration],
-    fetchSize: Int,
```

Comment:
Are we just keeping this unconfigured now - because I don't see anything setting `page-size`
https://docs.datastax.com/en/developer/java-driver/4.3/manual/core/paging/#paging

Reply:
:+1: